src/result_generation/height/height_mutiartifact.py

```diff
@@ -0,0 +1,57 @@
+import sys
+from datetime import datetime
+from functools import partial
+from pathlib import Path
+
+import numpy as np
+import tensorflow as tf
+
+from result_generation.height.height import HeightFlow
+
+sys.path.append(str(Path(__file__).parents[1]))
+import utils.inference as inference  # noqa: E402
+import utils.preprocessing as preprocessing  # noqa: E402
+
+
+class HeightFlowMutliArtifact(HeightFlow):
+
+    def process_depthmaps_depthmapmultiartifactlatefusion(self):
+        depthmaps_file = []
+        for artifact in self.artifacts:
+            input_path = self.get_input_path(
+                self.scan_directory, artifact['file'])
+            depthmaps_file.append(input_path)
```

Comment:
[nit, personal taste] This can be made shorter

```suggestion
        depthmaps_file = [self.get_input_path(self.scan_directory, artifact['file'])
                          for artifact in self.artifacts]
```